### PR TITLE
track overlay

### PR DESCRIPTION
**Pull request by robotlabs**
track overlay

### CODE REVIEW

File: src/components/confirm-send/index.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export default ({
   sendError,
   sending,
   sendComplete,
-  onModalHide
+  onModalHide,
+  trackSource
```
Comment (ianmcgregor) [inline src/components/confirm-send/index.js]: this prop isn't being used btw

Comment (robotlabs) [reply src/components/confirm-send/index.js]: RIGHT! well spotted, thanks

File: src/screens/buy-sell/screens/success-purchase/index.js [hunk 2]
```diff
@@ -6,9 +6,11 @@ import Title from 'app/components/title'
 import Paragraph from 'app/components/paragraph'
 import Icon from 'app/components/icons/icon'
 import moneyFormat from 'app/utils/money-format'
+import { track } from 'app/utils/track'
 
 export const SuccessPurchase = props => {
   const { totWollo, closeScreen } = props.navigation.state.params
+  track('OVERLAY : SUCCESS: PURCHASE', { type: 'success', title: 'success' })
```
Comment (ianmcgregor) [inline src/screens/buy-sell/screens/success-purchase/index.js]: Should it track on every render?

Comment (robotlabs) [reply src/screens/buy-sell/screens/success-purchase/index.js]: yes. it renders when it is being instantiated

**Pull request merged**